runtime/testy/studentAPI.py

```diff
@@ -61,17 +62,16 @@ class Gamepad(StudentAPI):
     "joystick_right_y" : 3
   }
 
-  def __init__(self):
-    super().__init__(self.toManager, self.fromManager)
+  def __init__(self, toManager, fromManager):
+    super().__init__(toManager, fromManager)
 
   def get_value(self, name, gamepad_number=0):
-    gamepad_dict = self._getSMValue('gamepads')[gamepad_number]
-    if name in self.joysticks.keys():
+    gamepad_dict = self._getSMValue("gamepads")[gamepad_number]
+    if name in self.joysticks:
       return gamepad_dict["axes"][self.joysticks[name]]
-    elif name in self.buttons.keys():
+    if name in self.buttons:
```

Comment:
I guess I personally find it simpler if we have `return` statements since there's no need for the `el` part.